data/World/Overworld.json

```diff
@@ -1570,12 +1773,30 @@
         "exits": {
             "DMC Upper Nearby": "True",
             "DMC Ladder Area Nearby": "True",
+            "DMC Pierre Platform": "
+                can_use(Goron_Tunic) and ((damage_multiplier != 'ohko' and damage_multiplier != 'quadruple') or 
+                (Fairy and not entrance_shuffle) or can_use(Nayrus_Love))",
             "DMC Central Nearby": "
                 can_use(Goron_Tunic) and can_use(Longshot) and 
                 ((damage_multiplier != 'ohko' and damage_multiplier != 'quadruple') or 
                     (Fairy and not entrance_shuffle) or can_use(Nayrus_Love))"
         }
     },
+    {
+        "region_name": "DMC Pierre Platform",
+        "font_color": "Red",
+        "scene": "Death Mountain Crater",
+        "hint": "Death Mountain Crater",
+        "locations": {
+            "DMC Adult Green Rupee 1": "here(is_adult)",
+            "DMC Adult Green Rupee 2": "here(is_adult)",
+            "DMC Adult Green Rupee 3": "here(is_adult)",
+            "DMC Adult Green Rupee 4": "here(is_adult)",
+            "DMC Adult Green Rupee 5": "here(is_adult)",
+            "DMC Adult Green Rupee 6": "here(is_adult)",
+            "DMC Adult Red Rupee": "here(is_adult)"
```

Comment:
Just is_adult. here() is not doing anything.

Reply:
Fixed.